Tools/autotest/arducopter.py

```diff
@@ -3832,8 +3894,11 @@ def tests(self):
              "Log download",
              lambda: self.log_download(
                  self.buildlogs_path("ArduCopter-log.bin"),
-                 upload_logs=len(self.fail_list) > 0))
+                 upload_logs=len(self.fail_list) > 0)),
 
+            ("MotorVibration",
```

Comment:
Needs to go before log download, not after

Reply:
ok